pyomo/core/base/var.py

```diff
@@ -735,7 +738,7 @@ class SimpleVar(_GeneralVarData, Var):
 
     def __init__(self, *args, **kwd):
         _GeneralVarData.__init__(self,
-                                 domain=Reals,
+                                 domain=None,
```

Comment:
What is the intent of this change?  If no domain is specified, then ...?

Reply:
Exactly.  It turns out that that initialization is a no-op.  I was banging my head as to why we were setting all scalar vars to be Reals.  The default value comes from the `Var.__init__` below.  I removed the `Reals` here to make it more obvious that *this* method was not responsible for setting the domain.